mobile/src/screens/missions/index.jsx

```diff
@@ -136,15 +128,18 @@ export default function MissionsScreen() {
                     </Popover>
                   ))}
                 </Flex>
-                <HStack alignItems="center">
-                  <FontAwesome name="hourglass" />
-                  <Text p={3} fontWeight="bold" color="danger.500">
-                    {item.expires_at
-                      ? formatTimeLeft(item.expires_at)
-                      : "take your time"}
-                  </Text>
-                </HStack>
               </Flex>
+              <HStack alignSelf="flex-end" alignItems="center">
+                <FontAwesome name="hourglass" />
+                <Text p={3} fontWeight="bold" color="danger.500">
+                  {item.expires_at
+                    ? formatDistanceToNow(new Date(item.expires_at), {
+                        addSuffix: true,
+                        locale: pt,
+                      })
+                    : "take your time"}
```

Comment:
"take your time" here means what?

Reply:
means that there is no expiration time for the quest, so the user can answer the question anytime and the quest will be always visible to be answered

Reply:
Make sense 👍